test/03.cache.js

```diff
@@ -0,0 +1,76 @@
+var assert = require('assert');
+var path = require('path');
+var expressBhEngine = require('../lib/engines/bh');
+var fs = require('fs');
+
+var bemjson = {
+    block: 'page',
+    title: 'Example'
+};
+var bundlePath = path.join(__dirname, 'data/views/pages/cache');
+var filePath = path.join(bundlePath, 'cache.bh.js');
+var templateContent = [
+'var BH = require(\'../../../../../node_modules/bh/lib/bh.js\');',
+'var bh = new BH();',
+'bh.match(\'page\', function (ctx) {',
+'    ctx.content(ctx.json().title);',
+'});',
+'module.exports = bh;'
+].join('\n');
+
+describe('cache', function () {
+
+    beforeEach(function () {
+        fs.appendFileSync(filePath, templateContent);
+    });
+
+    afterEach(function () {
+        fs.truncateSync(filePath, 0);
```

Comment:
I was not able to change this approach on a `mock-fs`.
I catched an error like:
`
Error: Cannot find module '../../../../../node_modules/bh/lib/bh.js'
`